### DISCUSSION TITLE
Backup when External HD Detected

### DISCUSSION THREAD

**Discussion by PeterKW** [borgbase/vorta#873] [FAQ] [answered]
Sorry if there's an obvious answer out there for this that I have missed. Is there a way of running a vorta profile backup when a particular external drive is mounted?

Thank you very much

**Comment by m3nu** [comment, marked as answer]
You can start a backup for a given profile from the command line, when the drive is plugged in. Specifics on where to put this command vary by OS. Would be nice to have docs about this.

**Reply by PeterKW** [reply]
Yeah that would be useful. Probably an obvious question but I haven't managed to find the documentation yet. Is there a way in the Vorta CLI to run a profile by name or do I need to run something from the Borg CLI somehow?
Thanks

**Reply by m3nu** [reply]
Yes, you can start a backup for a profile by name. I just added this to the docs, since it was only documented in the CLI args so far.

https://vorta.borgbase.com/usage/#trigger-backups-externally

```
usage: vorta [-h] [--version] [--daemonize] [--create PROFILE]

Vorta Backup GUI for Borg.

optional arguments:
  -h, --help        show this help message and exit
  --version, -V     Show version and exit.
  --daemonize, -d   Fork to background and don't open window on startup.
  --create PROFILE  Create a backup in the background using the given profile.
                    Vorta must already be running for this to work.
```
